packages/kit/src/core/route_preprocessor.js

```diff
@@ -0,0 +1,38 @@
+import path from 'path';
+import MagicString from 'magic-string';
+
+/**
+ * Injects __handle_route of `@sveltejs/init-route` to handle scrolling when component mounts
```

Comment:
I did Ctrl+f for `init-route` and don't see anything. Was it renamed or accidentally left off the PR?

Reply:
Oops yeah I left that one while refactoring some stuff. Should be `@sveltejs/kit/router`.

Reply:
```suggestion
 * Injects `use:__handle_route` from `@sveltejs/kit/router` to handle scrolling when route component mounts
```